app/connectors/httpParsers/SaveSubmittedProcessHttpParser.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors.httpParsers
+
+import models.errors.{InternalServerError, InvalidProcessError}
+import models.{RequestOutcome, SaveSubmittedProcessResponse}
+import play.api.Logger
+import play.api.http.Status._
+import uk.gov.hmrc.http.HttpReads
+
+object SaveSubmittedProcessHttpParser extends HttpParser {
+
+  val logger: Logger = Logger(SaveSubmittedProcessHttpParser.getClass)
```

Comment:
Although we were creating loggers using the containing class, a recent PRA recommends we go back using the Logger object directly. eg Logger.info("blah blah")

Reply:
This is a fairly recent change so some of the existing code needs updating also.

Reply:
code updated